internal/pkg/template/templates/workloads/partials/cf/instancerole.yml

```diff
@@ -38,6 +38,25 @@ InstanceRole:
                 StringEquals:
                   'iam:ResourceTag/copilot-application': !Sub '${AppName}'
                   'iam:ResourceTag/copilot-environment': !Sub '${EnvName}'
+      - PolicyName: !Join ['', [!Ref AppName, '-', !Ref EnvName, '-', !Ref WorkloadName, SecretsPolicy]]
```

Comment:
We might only want to include this additional policy if they have secrets enabled? aka the length of `EnvironmentSecrets` is not 0 anymore.

Reply:
How do I include this? Withinn if statements?
{{- if hasSecrets .}}
- PolicyName: !Join ['', [!Ref AppName, '-', !Ref EnvName, '-', !Ref WorkloadName, SecretsPolicy]]......
{{- end }}

Reply:
yes something like that if you use `hasSecrets` for rdws as well.

Reply:
Done. Updated in next commit.

Reply:
It's ok we'll squash and merge the PR after it gets approved, so don't worry about spamming commits.